server.js

```diff
@@ -78,7 +78,7 @@ const mainLocale = (selectedLocales && selectedLocales[0]) || 'en'
 
 // Initializes the server and mounts it in the generated build directory.
 http.createServer((req, res) => {
-  // If we are accessing the root, it should be redirected to `/en` instead.
+  // If we are accessing the root, it should be redirected to the default language,
```

Comment:
Unrelated to this patch, I just noticed it here. Maybe this should be `res.redirect`?

In another PR, just thinking out loud :)

Reply:
No. here I just mean that when DEFAULT_LOCALES is null or multiple languages, ONLY the first one will be taken instead of 'en'.
BTW, we don't have 'redirect' function for 'res' here, so change 'url' is also fine.